src/vip_client/classes/VipCI.py

```diff
@@ -70,7 +70,7 @@ class VipCI(VipLauncher):
                     #############
     def __init__(
         self, output_dir=None, pipeline_id: str=None, input_settings: dict=None, 
-        session_name: str=None, verbose: bool=None
+        session_name: str=None, verbose: bool=None, custom_wf_metadata: dict=None
```

Comment:
Please consider setting this new variableas a property to be saved: 
- add in variable `_PROPERTIES`
- maybe declare as a new main property on line 66 (see model in `VipLauncher`